include/mata/nfa-plumbing.hh

```diff
@@ -24,6 +24,12 @@ namespace Mata {
 
 namespace Nfa {
 
+/**
+ * Simplified NFA API, used in binding to call NFA algorithms.
+ *
+ * In particular, this mostly includes operations and checks, that do not return Automaton,
```

Comment:
Mostly? If there are only these kind of functions we can probably find a better name than `Plumbing`.

Reply:
Currently, I saw only these functions. I mean these are listed in doxygen.

Reply:
AFAIK, there are only these functions as of now, but the intent is to use `Mata::Plumbing` for other purposes as well (anything that we really do not want to recommend to the user to use unless they explicitly need these functions).